.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
```

Comment:
what about `docker`, `git submodules` (?), `github actions`?

Reply:
can add `docker` and `github-actions`, why `git submodules`?

Reply:
didn't check that. the first 2 should suffice